vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -7,154 +7,223 @@
 #include <numeric>
 #include <algorithm>
 #include <unordered_set>
+#include <unordered_map>
 
 using namespace LEARNER;
 using namespace VW;
 using namespace VW::config;
 
-struct ccb {};
+struct ccb {
```

Comment:
Currently the destructor is not called for the data object (only free), so you'll need to call them explicitly in the finish function.